stage 1 - A/Fire-Reservoir/Backend-template/targets/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+from .models import Target
+
+
+class TargetSerializer(serializers.ModelSerializer):
+    class Meta:
+        # define here your model and the fields to serialize
+        model = Target
+        fields = "__all__"
+
+    # not necessary. 
+    # def create(self, validated_data):
+    #     return super(TargetSerializer, self).create(validated_data)
+    
+    # not necessary.
+    # def update(self, instance, validated_data):
```

Comment:
it's nice and work it really not necessary but you need to implement this method 
see:
     # Implement here an update function